infra/terraform/platform/main.tf

```diff
@@ -60,46 +49,46 @@ module "user_nfs_softnas" {
 }
 
 module "data_backup" {
-  source = "../../modules/data_backup"
+  source = "../modules/data_backup"
 
-  env                 = "${var.env}"
-  k8s_worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/nodes.${var.env}.${data.terraform_remote_state.base.xyz_root_domain}"
+  env                 = "${terraform.workspace}"
+  k8s_worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/nodes.${terraform.workspace}.${data.terraform_remote_state.base.xyz_root_domain}"
   logs_bucket_arn     = "${data.terraform_remote_state.base.s3_logs_bucket_name}"
 }
 
 module "encrypt_scratch_lambda_function" {
-  source     = "../../modules/lambda_functions"
-  env        = "${var.env}"
+  source     = "../modules/lambda_functions"
+  env        = "${terraform.workspace}"
   bucket_id  = "${module.data_buckets.scratch_bucket_id}"
   bucket_arn = "${module.data_buckets.scratch_bucket_arn}"
 }
 
 module "encrypt_crest_lambda_function" {
-  source     = "../../modules/lambda_functions"
-  env        = "${var.env}"
+  source     = "../modules/lambda_functions"
+  env        = "${terraform.workspace}"
   bucket_id  = "${module.data_buckets.crest_bucket_id}"
   bucket_arn = "${module.data_buckets.crest_bucket_arn}"
 }
 
 module "container_registry" {
-  source = "../../modules/container_registry"
-  env    = "${var.env}"
+  source = "../modules/container_registry"
+  env    = "${terraform.workspace}"
 }
 
 module "federated_identity" {
```

Comment:
I just noticed that we still have the `modules/federated_identity/saml` directory, it may be a good opportunity to delete it.

I mean, it's still committed and we can get it back if we need to, but I don't think it's used anywhere.